deploy/task-definition-template.json

```diff
@@ -19,6 +19,22 @@
         {
           "name": "RAILS_SERVE_STATIC_FILES",
           "value": "true"
+        },
+        {
+          "name": "MATOMO_URL",
+          "value": "thttps://analytics.buildkite.com"
+        },
+        {
+          "name": "MATOMO_SITE_ID",
+          "value": "1"
```

Comment:
@toolmantim I cargo-culted these values from elsewhere. Is this site ID appropriate for docs?